### DISCUSSION TITLE
Increasing number of overgoal copies

### DISCUSSION THREAD

**Discussion by stevew-Purdue** [moosefs/moosefs#417] [Q&A]
Hi,

I have a MooseFS installation with nine chunkservers, all running v. 3.0.114, and a total raw space of 2.0 PiB.  The goal for the entire filesystem is set at two copies.  At the moment, seven chunkservers are at 95.46% used, one is at 96.43% and one at 96.46%.  For the past week or so the number of overgoal copies has been steadily increasing to where I now have 1.4M overgoal copies (28.5M of "at goal" copies).

The only unusual things that have happened (that I can think of) in the past few weeks were:
    1) a couple of hard drives failed and needed to be pulled out.
    2) a number of disks were moved from one chunkserver to a newly commissioned chunkserver

Does anyone have any idea of what's going on and how to correct this?

Thanks!
Steve

**Comment by chogata** [comment]
A question: do you still have any disks with problems in your instance? Check "Disks" tab in your CGI, column "last error", is there any disk that shows a date instead of "no errors"? Or just any disk with avg read/write times a lot bigger than the other disks? If yes, you might be suffering from the lock issue upon i/o errors that was corrected in 3.0.115. Upgrade will help you.

**Comment by stevew-Purdue** [comment]
Oh, that might be it!  Two of the chunkservers (the two with a higher percentage of space usage) have reported disk errors but with a current OK status.  I'll try upgrading the chunkservers to see if that corrects the problem.

Steve

**Comment by stevew-Purdue** [comment]
I completed the upgrade to 3.0.115 and that appears to have corrected the problem.  Thanks for your help!

Steve
